src/_main_/utils/constants.py

```diff
@@ -91,3 +91,5 @@
     "Join the green revolution and make a difference."
 ]
 
+LANGUAGE_CODES_TO_NOT_SPLIT = { "en-GB", "zh-CN", "zh-TW", "mni-Mtei" }
```

Comment:
Sorry about the large PR. I pushed my updates too late. I'll look into spliting it, if it can be done after the fact by some mean or maybe find another way round it.
I'm working on adding the tests at the moment.